src/lib.rs

```diff
@@ -206,6 +206,7 @@ impl<T> Counter for T
 /// = 2048 * 2^(k-1)`, which is the k-1'th bucket's end. So, we would use the previous bucket
 /// for those lower values as it has better precision.
 ///
+#[derive(Debug)]
```

Comment:
Hmm, I wonder if we'd rather implement `Debug` ourselves. I feel like the autogenerated one won't be particularly helpful (or readable).